guide/inflation.rst

```diff
@@ -151,72 +262,140 @@ In the namelist each entry has two values. The first is for Prior inflation and
-``inf_damping`` > 0.). Then enable Prior state space, spatially-varying inflation, with no Posterior inflation (set
-``inf_flavor`` to [2, 0]). Then try damped inflation (set ``inf_damping`` to 0.9 and set ``inf_sd_initial`` and
-``inf_sd_lower_bound`` to 0.6). The inflation values and standard deviation are written out to files with
-``_{prior,post}inf_{mean,sd}{_d##}.nc``, in their names
-(d\#\# is the domain number, if needed). These NetCDF files can be viewed with common tools (we often use
-`ncview <http://meteora.ucsd.edu/~pierce/ncview_home_page.html>`__ ). Expected inflation values are generally in the 1
-to 30 range; if values grow much larger than this it usually indicates a problem with the assimilation.
-
-:doc:`../assimilation_code/programs/fill_inflation_restart/fill_inflation_restart` 
-may be used to create netCDF files with initial values such that the 
-input.nml settings for reading from file vs. reading from namelist can stay constant 
-throughout the entire experiment.
-
-It is possible to set inflation values in an existing netCDF file by using one of the standard NCO utilities like
-"``ncap2``" on a copy of a restart file. Inflation mean and sd values look exactly like restart values, arranged by
-variable type like T, U, V, etc.
-
-Here's an example of using ncap2 to set the T,U and V inf values:
-
-.. container:: unix
-
-   ::
-
-        ncap2 -s 'T=1.0;U=1.0;V=1.0' wrfinput_d01 input_priorinf_mean_d01.nc
-        ncap2 -s 'T=0.6;U=0.6;V=0.6' wrfinput_d01 input_priorinf_sd_d01.nc
-        -or-
-        ncap2 -s 'T(:,:,:)=1.0;U(:,:,:)=1.0;V(:,:,:)=1.0' wrfinput_d01 input_priorinf_mean_d01.nc
-        ncap2 -s 'T(:,:,:)=0.6;U(:,:,:)=0.6;V(:,:,:)=0.6' wrfinput_d01 input_priorinf_sd_d01.nc
-
-Some versions of the NCO utilities change the full 3D arrays into a single scalar. If that's your result (check your
-output with ``ncdump -h``) use the alternate syntax or a more recent version of the NCO tools.
+it possible to restart an experiment. For many of the supported models, the scripting that
+does the renaming and copying of the inflation files may already be provided.
+
+.. Important::
+		Inflation is only be applied to the variables that are **updated** by DART
+		by specifying ``UPDATE`` within the ``&model_nml`` section of ``input.nml``.
+		If the variables are not updated (``NO_COPY_BACK``), then inflation should
+		not impact them. ``NO_COPY_BACK`` variables are often used to compute the forward
+		operators, however, they don't take part in the update and as such
+		they should not be inflated.
+
+		In order to make sure a variable with a ``NO_COPY_BACK`` tag is not inflated, the
+		user needs to run ``fill_inflation_restart`` and assign an inflation value of 1
+		to that variable. Additional details on how to run the ``fill_inflation_restart``
+		program can be found in the "*Caution*" section below.
```

Comment:
Suggest this to be clearer that users have to set inflation to 1 to get no inflation. 

```suggestion
		Inflation should only be applied to the variables that are **updated** by DART.	
		
		Some models have `NO_COPY_BACK`` variables which are often used to compute the forward
		operators, however they should not be inflated. Whether a variable is `NO_COPY_BACK`, or  `UPDATE`
		is set in the ``&model_nml``. 

		In order to make sure a variable with a ``NO_COPY_BACK`` tag is not inflated, the
		user needs to run ``fill_inflation_restart`` and assign an inflation value of 1
		to that variable. Additional details on how to run the ``fill_inflation_restart``
		program can be found in the "*Caution*" section below.
```